src/scripts.js

```diff
@@ -28,16 +26,51 @@ const ingredientSidebar = document.querySelector("#ingredientSection")
 const userName = document.querySelector('#user-info');
 const favoritesView = document.querySelector('#favorites-view');
 const savedButton = document.querySelector('#saved-recipe-button');
+const totalCost = document.querySelector('#totalCost')
+const ingredientList = document.querySelector('#ingredientList');
+const ingredientAmounts = document.querySelector('#ingredientAmounts')
+
+//********WIP
+let radioButtons = document.querySelectorAll('.food-category');
+let submitTagButton = document.querySelector("#submitTagButton");
 
 // ~~~~~~~~~~~~~~ Event Listeners ~~~~~~~~~~~~~~~~~~~~
 allRecipes.addEventListener('click', viewRecipeDetail);
 window.addEventListener('load', displayAllRecipes);
 window.addEventListener('load', displayWelcomeMessage);
 allRecipes.addEventListener('click', addRecipeToFavorites);
 savedButton.addEventListener('click', displayFavorites);
+submitTagButton.addEventListener('click', displayFilteredTag);
 
 // ~~~~~~~~~~~~~~ Functions ~~~~~~~~~~~~~~~~~~~~
 
+function checkTagType(){
+    let messageType = "";
+
+    radioButtons.forEach((currentRadioButton) => {
+        if(currentRadioButton.checked){
+            messageType = currentRadioButton.value;
+        }
+    })
+    return messageType;
+}
+
+function displayFilteredTag(){
+    const tagSelected = checkTagType();
+    const tagSelectedList = recipeRepository.filterTag(tagSelected)
+
+    allRecipes.innerHTML = ""
+
+    if(tagSelected === "all"){
```

Comment:
what is the if(tagSelected ==='all')? Is this if you select all the radio buttons it displays all the recipes?